components/automate-ui/src/app/modules/infra-proxy/chef-servers-list/chef-servers-list.component.html

```diff
@@ -36,11 +36,11 @@
         <chef-table>
           <chef-thead>
             <chef-tr>
-              <chef-th>Name</chef-th>
-              <chef-th>FQDN</chef-th>
-              <chef-th>IP Address</chef-th>
-              <chef-th>No Of Orgs</chef-th>
-              <chef-th class="three-dot-column"></chef-th>
+              <chef-th  class="no_border_th">Name</chef-th>
+              <chef-th class="no_border_th">FQDN</chef-th>
+              <chef-th class="no_border_th">IP Address</chef-th>
+              <chef-th class="no_border_th">No Of Orgs</chef-th>
+              <chef-th class="three-dot-column  no_border_th"></chef-th>
             </chef-tr>
```

Comment:
can we follow either Kebab case  or snake case. Kebab case will be best because we are using same pattern in other place